### DISCUSSION TITLE
Trainer port outputs should be RAW stick movements

### DISCUSSION THREAD

**Discussion by ror1948** [EdgeTX/edgetx#1727] [General]
Hardware: 
1) TX16S with AETR Edgetx 2.5 and Multi 1.3.3.7
2) TX16S with AETR OpenTx 2.3.14 and Multi 1.3.3.7

Produce the problem:
-Create one Tx as Master with ch1 inverted
-Create other Tx as Slave set with ch1 inverted
-in other words both Tx have the same model setup so either Tx could be the master to the model.
- create switch on master to activate trainer mode.
- when in trainer mode using SLAVE the ch1 is inverted as seen on the model control surface.
- when trainer mode is OFF using Master Tx, ch1 is correct as seen on the model control surface.

The trainer outputs on the trainer cable should be the RAW values of the sticks and not be processed by the output setup of model.

**Comment by raphaelcoeffic** [comment]
@ror1948 the output have been used since ever. What would you prefer? The Stick value? The input value? the Mixer values? Everyone has a different use-case. Fortunately, you could make a couple more channels (above those used by the receiver) with exactly these raw values, and select these channels to be sent.
